src/main/java/com/google/sps/servlets/FetchAnswersServlet.java

```diff
@@ -35,15 +38,90 @@
 /** 
  * Retrieves the answers to be displayed on the page.
  */
-@WebServlet("/answers")
+@WebServlet("/fetch-answers")
 public class FetchAnswersServlet extends HttpServlet {
 
   /** 
-   * Gets the answers for a single question and send them back as JSON.
-   * TODO(shaargtz): Modify to get style and functionality right.
+   * This method will get the answers for a single question and send them back as JSON.
```

Comment:
nit: Use simple present tense.
```suggestion
   * Gets the answers for a single question and send them back as JSON.
```

Reply:
done